src/test/resources/org/zalando/nakadi/validation/invalid-json-schema-examples.json

```diff
@@ -147,9 +147,7 @@
         }
       }
     },
-    "errors": ["Invalid schema found in [#/definitions/paradox]: extraneous key [not] is not permitted",
-      "Invalid schema found in [#/items]: extraneous key [$ref] is not permitted",
```

Comment:
It's interesting that you actually had a test which checks that $ref is forbidden 😄